packages/astro/src/cli/install-package.ts

```diff
@@ -161,3 +162,56 @@ async function installPackage(
 		return false;
 	}
 }
+
+export async function fetchPackageJson(
```

Comment:
Fished this out of `astro add`, code is unchanged. `astro add` is insanely big so fishing things out of it is honestly just good.